modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
Umm I am worried about this guy conflicting with the browser [Node API](
https://developer.mozilla.org/en/docs/Web/API/Node).


On Thu, Jun 16, 2016 at 9:20 AM Bryan Housel <notifications@github.com>
wrote:

> In modules/core/index.js
> <https://github.com/openstreetmap/iD/pull/3174#discussion_r67283776>:
>
> > @@ -0,0 +1,10 @@
> > +export { Connection } from './connection';
> > +export { Difference } from './difference';
> > +export { Entity } from './entity';
> > +export { Graph } from './graph';
> > +export { History } from './history';
> > +export { iD_Node as Node } from './node';
>
> Why is this one iD_Node? Can it be just Node to be consistent with Way
> and Relation?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openstreetmap/iD/pull/3174/files/80159d158940f15baa3bd063867d3f44d80e0eea#r67283776>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGpL7tpPz8jBpDNHOXcdL-xZqkqDbba8ks5qMMfxgaJpZM4I2wfj>
> .
>